load_env.py

```
@@ -2,12 +2,32 @@
+import re
 from pathlib import Path
 from typing import Optional
 
 _PACKAGE_ROOT = Path(__file__).resolve().parent
 _ENV_PATH = _PACKAGE_ROOT / ".env"
 
+_QUOTED_HASH_PATTERN = re.compile(r'''(?P<in_double_quotes>"[^"]*")|(?P<in_single_quotes>'[^']*')|(?P<comment>#[^\n]*)''')
+
+
+def _strip_trailing_comment(value: str) -> str:
+    """移除值末尾的行内注释，但保留引号内的 # 字符。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)